packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -21,6 +21,7 @@
   min-inline-size: 140px;
 
   .container {
```

Comment:
@eriklharper Once merged, can you help consolidate styles in https://github.com/Esri/calcite-design-system/pull/8594/files#diff-e5512e4c4c1a21e32c7ace6b5d5a0dbf74427018fa79efbd26076e6a93cc3eda?

Reply:
PR ready for review here: https://github.com/Esri/calcite-design-system/pull/9012